golang/outbound.go

```diff
@@ -56,12 +56,17 @@ func (c *Connection) beginCall(ctx context.Context, serviceName string) (*Outbou
 		return nil, err
 	}
 
+	headers := callHeaders{
+		CallerName: c.localPeerInfo.ProcessName,
```

Comment:
Ok cool, I've added a serviceName string to the TChannel constructor, and I use that in the cn header now.